src/components/Select.js

```diff
@@ -156,6 +217,11 @@ const styles = {
   selected: {
     position: 'relative'
   },
+  activeItem: {
+    backgroundColor: StyleConstants.Colors.PRIMARY,
+    color: StyleConstants.Colors.INVERSE_PRIMARY,
```

Comment:
I'm updating the colors in another PR, to avoid conflict we'll want to change this to `WHITE` instead of `INVERSE_PRIMARY`.